packages/firestore/src/core/sync_engine.ts

```diff
@@ -495,13 +500,11 @@ export class SyncEngine implements RemoteSyncer, SharedClientStateSyncer {
       );
       return this.applyRemoteEvent(event);
     } else {
-      const queryView = this.queryViewsByTarget[targetId];
-      assert(!!queryView, 'Unknown targetId: ' + targetId);
+      assert(!!this.queriesByTarget[targetId], 'Unknown targetId: ' + targetId);
```

Comment:
In general, we only want to add assert that are needed for the client to function properly. Since `this.queriesByTarget[targetId]` is not accessed here, you can drop the assert.

Reply:
Done.